src/dist/block_service/fds/fds_service.cpp

```diff
@@ -95,10 +98,42 @@ DEFINE_THREAD_POOL_CODE(THREAD_POOL_FDS_SERVICE)
 DEFINE_TASK_CODE(LPC_FDS_CALL, TASK_PRIORITY_COMMON, THREAD_POOL_FDS_SERVICE)
 
 const std::string fds_service::FILE_LENGTH_CUSTOM_KEY = "x-xiaomi-meta-content-length";
-const std::string fds_service::FILE_LENGTH_KEY = "content-length";
 const std::string fds_service::FILE_MD5_KEY = "content-md5";
 
-fds_service::fds_service() {}
+fds_service::fds_service()
+{
+    const int BYTE_TO_BIT = 8;
+
+    // get max sst file size in normal scenario
+    uint64_t target_file_size =
```

Comment:
These code is some kind of tricky to read configs of 'pegasus.server', and some threshold is not very strict. Add some comments to explain would be better.